README.md

```diff
@@ -53,7 +53,7 @@ _NOTE: set the fetch-depth for `actions/checkout@v2` to be sure you retrieve all
 #### Outputs
 
 * **new_tag** - The value of the newly created tag.
-* **tag** - The value of the latest tag after running this action.
+* **tag** - The value of the latest tag before running this action.
```

Comment:
This is originally intended to be the tag that is a result/output of this action (i.e. the new tag). Is that not the case? Otherwise why the change?